libs/libc/string/lib_memrchr.c

```diff
@@ -26,6 +26,34 @@
 
 #include <string.h>
 
+/****************************************************************************
```

Comment:
code with a different license than the one in the header should not be imported in the same file.

Reply:
Got the newlib author response, see the Summary